lib/models/God.js

```diff
@@ -0,0 +1,54 @@
+const pool = require('../utils/pool');
+
+class God {
+  id;
+  name;
+  known;
+  drink;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.known = row.known;
+    this.drink = row.drink;
+  }
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * from gods');
+    return rows;
+  }
+  static async getById(id) {
+    const { rows } = await pool.query('SELECT * from gods WHERE id=$1', [id]);
+    return rows[0];
+  }
```

Comment:
This could be the false 200 status we are receiving, let's run the rows[0] through the constructor => return new God(rows[0])